### ISSUE TITLE
F-droid version is not the last one

### ISSUE THREAD

**Issue by edaubert** [rootkiwi/an2linuxclient#24]
Hi,

It seems that the version on f-droid is still 0.7.0 while the version on Google Play is 0.8.0.

I got an issue with the 0.7.0 but the issue seems to be fixed on 0.8.0.

Is there any reason which explain this difference ?

Anyway this app is really usefull. Thanks a lot !

**Comment by rootkiwi** [comment]
Hello, I'm not sure why. If you look here:
https://gitlab.com/fdroid/fdroiddata/blob/master/metadata/kiwi.root.an2linuxclient.txt

This probably have something to do with it, if someone knows what the problem is please tell:
> Build:0.8.0,10
>     disable=lintVitalRelease fails

I'm glad you find it useful!

**Comment by edaubert** [comment]
As far as I understand the documentation given here: https://f-droid.org/en/docs/Build_Metadata_Reference/#Disabled

`disable=lintVitalRelease fails` disable the build of the v0.8.0 version which may explain why this version is not available on f-droid.

The question though is why the build has been disabled. The message says that it is related to a 'step' called `lintVitalRelease` which fails to complete.

According to what I have found about there is a task `lintVitalRelease` with graddle which gives me 60 errors and a failure status. Most of the errors are related to translation and Spanish and/or Dutch languages.

So I guess this is the issue.

I saw a commit about German translation in the master branch. However the errors on master grows to 146.

A 'simple' way to cope with this issues, according to graddle logs is:
```
lintOptions {
          checkReleaseBuilds false
          // Or, if you prefer, you can continue to check for errors in release builds,
          // but continue the build even when errors are found:
          abortOnError false
      }
```

But this means that the translation could be erroneous or incomplete.

I guess the proper way of dealing with this is to fix the translation issues but I don't speak German, neither Dutch nor Spanish...
